packages/common/e2e/catalog-collection.e2e.ts

```diff
@@ -88,6 +88,57 @@ describe("catalog and collection e2e:", () => {
       );
       const response = await instance.searchItems("colorado");
     });
+    describe("Catalog.contains:", () => {
+      it("located item by id", async () => {
```

Comment:
These are just used to validate behavior against the live api _before_ writing unit tests